src/main/resources/META-INF/rewrite/mockito.yml

```diff
@@ -147,3 +147,4 @@ recipeList:
       oldGroupId: org.mockito
       oldArtifactId: mockito-all
       newArtifactId: mockito-core
+  - org.openrewrite.java.testing.junit5.MockitoJUnitAddMockitoSettingsLenientStrictness
```

Comment:
As indicated, we'd only want to run this recipe if folks are in the process of moving from Mockito 1 or 2 on JUnit 4 to JUnit 5, as that's where the default looks to have changed [according to this answer](https://stackoverflow.com/a/53234137/53444). In general the docs are still a bit vague as to strictness, as seen from the many issues linked here:
- https://github.com/mockito/mockito/issues/2532

Here's an example PR that adds conditional logic for running a recipe only when still on an older version of a dependency; that should help restrict this run if we stick to a detached recipe.
- https://github.com/openrewrite/rewrite-spring/pull/555